code/game/objects/items/crab17.dm

```diff
@@ -29,11 +29,11 @@
 	icon = 'icons/obj/money_machine.dmi'
 	icon_state = "bogdanoff"
 	layer = TABLE_LAYER //So that the crate inside doesn't appear underneath
-	armor = list("melee" = 80, "bullet" = 50, "laser" = 50, "energy" = 100, "bomb" = 100, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 80)
+	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 60, "bomb" = 90, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 80)

```

Comment:
Don't touch melee.

I'll accept lower everything else.

Reply:
don't touch acid either actually, being able to press 100u a few times on a chem dispenser shouldn't mean you get to delete the distraction tool.

Reply:
i never touched acid

Reply:
Sorry my acid comment was for "I'll accept lower everything else."